sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/AnomalyFeedback.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The generated code exposes a:
```cs
public Guid? AnomalyDetectionConfigurationId { get; set; }
```

I think we can hide this one and extract the ID directly from:

```cs
public MetricAnomalyDetectionConfiguration AnomalyDetectionConfigurationSnapshot { get; set; }
```

Reply:
Do you mean hide the `AnomalyDetectionConfigurationId` and not replace it with a top level Id property?

Reply:
I think you can ignore this comment for the time being. It's probably just not very clear to me how these properties will be used yet. I was expecting the same a ID to be defined in `AnomalyDetectionConfigurationSnapshot`.

By the way, it seems `AnomalyDetectionConfigurationSnapshot` should be read-only, according to the service.